.gh-issues/issue-60-left-menu-popups-no-click-outside-close.md

```
@@ -0,0 +1,43 @@
+## Summary
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)